docs/reference-guide/network-ports.rst

```diff
@@ -0,0 +1,49 @@
+Network Ports
+=============
+
+etcd Ports
```

Comment:
I don't really like these titles. They refer to Ansible *groups*, but that isn't clear at all. Needs some introduction, at least.